lms/static/sass/course/_profile.scss

```diff
@@ -179,6 +179,58 @@
       width: 100%;
     }
 
+    > .credit-eligibility{
+      border-top: 1px solid $lightGrey;
+      margin-top: lh();
+      padding-left: 0;
```

Comment:
Please add RTL support (see this file for mixins https://github.com/edx/edx-platform/blob/master/lms/static/sass/vendor/bi-app/_mixins.scss) as I commented before.